demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -206,39 +288,15 @@ def from_path(path: Path) -> Optional["BaseContent"]:
             logger.error(f"Could not parse content item from path: {path}")
             return None
         try:
-            return model.from_orm(content_item_parser)
+            obj = model.from_orm(content_item_parser)
+            obj.git_status = git_status
```

Comment:
let's go back to `return model.from_orm(content_item_parser)`, and set the git_status in the first `if` condition